src/super_context.ml

```diff
@@ -160,26 +166,31 @@ let create
       | None -> Path.relative context.ocaml_bin "ocamlopt"
       | Some p -> p
     in
+    let open Action.Var_expansion in
+    let open Action.Var_expansion.Concat_or_split in
     let make =
       match Bin.make with
-      | None   -> "make"
-      | Some p -> Path.to_string p
+      | None   -> Paths ([Path.of_string "make"], Split)
```

Comment:
This needs to be Strings, otherwise jbuilder is going to think it's the file "make" in the current dir and expand it to ./make